compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -1435,7 +1435,6 @@ impl<F: AcirField> AcirContext<F> {
             name,
             BlackBoxFunc::MultiScalarMul
                 | BlackBoxFunc::Keccakf1600
-                | BlackBoxFunc::Sha256Compression
```

Comment:
With this flag activated for sha compression I was hitting `ASSERT(is_valid_variable(variable_index))` in bb for the new test case @guipublic

Reply:
This looks to also be an issue with the old more inefficient sha256_var.